storybook/stories/Input.stories.js

```diff
@@ -81,8 +81,7 @@ export const Inputs = () => {
         <Input
           label="Email"
           autoFocus={true}
-          value=""
-          onChangeText={() => {}}
+          // value=""
```

Comment:
The `value` and `onChangeText` are required props. So we can add them back right otherwise we will shown the propType failed warning.